src/components/Search/SearchFiltersParticipantsSelector.tsx

```diff
@@ -104,17 +104,21 @@ function SearchFiltersParticipantsSelector({initialAccountIDs, onFiltersUpdate}:
             true,
         );
 
-        const isCurrentUserSelected = selectedOptions.find((option) => option.accountID === chatOptions.currentUserOption?.accountID);
+        const currentUserSelected = formattedResults.section.data.find((option) => option.accountID === chatOptions.currentUserOption?.accountID);
```

Comment:
Why are finding the `currentUserSelected` before the if condition? Its only use AFAIK is inside the if condition. @FitseTLT